Sources/SwiftSyntax/TokenKind.swift.gyb

```diff
@@ -166,6 +166,28 @@ public enum RawTokenKind {
 %   if token.text:
     case .${token.swift_kind()}: return "${token.text}"
 %   end
+% end
+    default: return nil
+    }
+  }
+
+  @_spi(RawSyntax)
+  public init?(keyword text: SyntaxText) {
```

Comment:
What is this used for?

CC @fwcd We could maybe use this to make `TokenSyntax` conform to `ExpressibleByStringLiteral`.

Reply:
This is currently not used not necessary for this PR. Should I make a separate PR?

Reply:
Yes, let’s move this to the PR that actually starts using it.

Reply:
Removed